sb/robot/vision.py

```diff
@@ -0,0 +1,20 @@
+from collections import namedtuple
+
+# Points
+# TODO: World Coordinates
+PolarCoord = namedtuple("PolarCoord", "distance_meters rot_y_rad rot_y_deg")
+CartCoord = namedtuple("CartCoord", "x y z")
+
+# Marker class
+MarkerBase = namedtuple("Marker",
+                        "id size polar pixel_centre cartesian is_wall_marker is_token_marker")
+
+
+class Marker(MarkerBase):
```

Comment:
If all this class is doing is adding a `__str__`, I don't think it's necessary

Reply:
Fixed by using the Python 3.5 NamedTuples https://github.com/sourcebots/simulator/pull/1/commits/df207e06ee5baad65b5abb7139c79dbe616407b6